api/provider/aws/deploy_delete_test.go

```diff
@@ -39,7 +38,7 @@ func TestDeploy_deleteDeployTags(t *testing.T) {
 		}
 	}
 
-	if err := deploy.deleteDeployTags("deploy_id"); err != nil {
+	if err := deleteEntityTags(tagStore, "deploy", "deploy_id"); err != nil {
```

Comment:
I don't think we need this test anymore since it's just testing `deleteEntityTags`. I assume we already have a `TestDeleteEntityTags`?

Reply:
I'm fine w removing this but I'm not sure what else to target in this test otherwise

Reply:
the whole test hsould be removed

Reply:
the whole test should be removed